### DISCUSSION TITLE
Implement Sourcehut bridge

### DISCUSSION THREAD

**Discussion by skyfaller** [git-bug/git-bug#394] [Feature & Ideas]
[Sourcehut](https://sourcehut.org/) is still in alpha, so it might be a moving target, but it would be great if we could somehow bridge to [their ticket-tracking system](https://man.sr.ht/todo.sr.ht/).

**Comment by MichaelMure** [comment]
API is here: https://man.sr.ht/todo.sr.ht/api.md

**Comment by 6543** [comment]
instead of what you did with jira, I'm also interested to have a go-lib to Sourcehut for the Gitea migration tool (https://github.com/go-gitea/gitea/issues/8689)

does somebody actually have started creating one?

**Comment by skyfaller** [comment]
Sorry if this is an annoying question, but why did this end up in "discussions" while other bridge proposals like for Gitea are in "issues"? https://github.com/MichaelMure/git-bug/issues/349  Is this a statement about priority?

**Reply by MichaelMure** [reply]
It's just a statement of "things are not sorted out consistently". Priority is based on "someone is doing it".

**Comment by mcepl** [comment]
Also, I should mention here [this discussion on todo.sr.ht email list](https://lists.sr.ht/~sircmpwn/sr.ht-discuss/%3CCYLGXWF4VOED.3FM8RG6W8JRM5%40cepl.eu%3E).

**Comment by sudoforge** [comment]
Closing in favor of #1024, which is younger than this, but is already an issue.
